editor/plugins/animation_tree_editor_plugin.cpp

```diff
@@ -129,6 +130,11 @@ void AnimationTreeEditor::edit_path(const Vector<String> &p_path) {
 	} else {
 		current_root = ObjectID();
 		edited_path = button_path;
+
+		for (int i = 0; i < editors.size(); i++) {
+			memdelete(editors[i]);
+		}
+		editors.clear();
```

Comment:
```suggestion
		for (int i = 0; i < editors.size(); i++) {
			editors[i]->edit(Ref<AnimationNode>());
			editors[i]->hide();
		}
```
I did some more digging and it seems that hide() is fine for the editors, sorry. But before that, it should initialize it by setting an empty resource.